internal/progress.go

```diff
@@ -83,3 +96,19 @@ func (p *Progress) report(firstCall bool) {
 		fmt.Printf("\n")
 	}
 }
+
+func (p *Progress) reportFraction(firstCall bool) {
+	if p.verbose {
+		fmt.Printf("%s: %d/%d\n", p.message, p.progress, p.total)
+		return
+	}
+	if firstCall {
+		fmt.Printf("%s: %d/%d", p.message, p.progress, p.total)
+	} else {
+		delStr := strings.Repeat("\b", len(strconv.Itoa(int(p.progress-1)))+1+len(strconv.Itoa(int(p.total))))
```

Comment:
I am not sure what this is doing but my best guess is that this is a way to overwrite progress bar instead of printing on a new line on the terminal. Either way it's probably worth adding a comment.